assets/stylesheets/components/_global-header.scss

```diff
@@ -4,7 +4,11 @@
 
 .global-header--inner {
   @extend %content-block;
-  padding: 24px;
+  padding: ($baseline-grid-unit * 4);
```

Comment:
Yes, good point. TBH I was going to suggest we deal with this as part of a grid PR or similar?